client/src/pages/SignupUser.js

```diff
@@ -1,9 +1,21 @@
 import React from "react";
+import PreNavBar from "../Components/PreNavBar";
+import UserSignUp from "../Components/UserSignUp";
+import Footer from "../Components/Footer";
 
 function SignupUser() {
   return (
-    <div></div>
+    <div>
+
+      <PreNavBar />
+      <UserSignUp />
+      <Footer />
```

Comment:
Please remove the <PreNavBar /> and the <Footer /> from this page as both those components live in the top level of App.js in order to not get rerendered every time we change pages.